apps/communications/contacts/js/contacts.js

```diff
@@ -648,6 +648,7 @@ var Contacts = (function() {
       '/shared/js/text_normalizer.js',
       SHARED_UTILS_PATH + '/status.js',
       '/shared/js/contacts/utilities/dom.js',
+      '/contacts/js/match_service.js',
```

Comment:
The `MatchService` is used in form.js and detail.js, should we lazyload it just when those views are loaded?